include/rogue/protocols/rssi/Header.h

```diff
@@ -49,7 +49,7 @@ namespace rogue {
             public:
 
                //! Header Size
-               static const uint32_t HeaderSize = 8;
+               static const  int32_t HeaderSize = 8;
```

Comment:
Why changed from uint32 to int32?

Reply:
Because it is passed to an arg which takes a relative adjustment: 

 frame->getBuffer(0)->adjustHeader(-rpr::Header::HeaderSize);

Reply:
Wanted to avoid singed/unsigned confusion on that line.